nodejs/week3/calculatorRouter.js

```diff
@@ -0,0 +1,70 @@
+
+const express = require("express");
+const calculatorRouter = express.Router();
+
+// /calculator/addition?firstParam=1&secondParam=2 should respond with 3.
+
+calculatorRouter.get('/addition', (request,response) => {
+
+  console.log(request.query);
+  if (request.query.firstParam === {num1}, request.query.secondParam === {num2} ) { 
+
+  // New app using express module 
+  const num1 = parseInt(request.params.num1);
+  const num2 = parseInt(request.params.num2);
+  
+  var result =  num1 + num2;
+  
+  response.send(result); 
+};});
+
+// /calculator/subtration?firstParam=4&secondParam=2 should respond with 2.
+
+calculatorRouter.get('/subtration', (request,response) => {
+  console.log(request.query);
+  if (request.query.firstParam === {num1}, request.query.secondParam === {num2} ) { 
+
+  // New app using express module 
+
+  const num1 = parseInt(request.params.num1);
+  const num2 = parseInt(request.params.num2);
+  
+  var result =  num1 - num2;
+  
+  response.send(result); 
+};});
+
+// /calculator/multiply?firstParam=4&secondParam=2 should respond with 8.
+
+calculatorRouter.get('/multiply', (request,response) => {
+  console.log(request.query);
+  if (request.query.firstParam === {num1}, request.query.secondParam === {num2} ) { 
+
+
```

Comment:
Whe you have multiple line breaks like this, it makes me wonder why that is the case. Any specific reason for it?